core/include/dt-bindings/input/linux-event-codes.h

```diff
@@ -0,0 +1,864 @@
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
```

Comment:
We may have a problem with this (no BSD-compatible license)

Reply:
ditto.